src/containers/BooksList.js

```diff
@@ -2,41 +2,59 @@ import { nanoid } from 'nanoid';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
+import CategoryFilter from '../components/CategoryFilter';
+import { changeFilter } from '../actions/index';
 
 function BooksList(props) {
-  const { books } = props;
+  const { books, category, handleFilterChange } = props;
+
```

Comment:
__[Optional]__ Destructure props from the function parameters in functional components to avoid additional variable declarations. e.g.

```javascript
// This way uses an extra line of code and makes the component more verbose
const MyComponent = (props) => {
    const { name, age, height } = props;
    ...
}

// This way achieves the same result with less code in a more reasonable way
const MyComponent = ({ name, age, height }) => {
  ...
}
```